client/layout/masterbar-item-notifications.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import MasterbarItem from './masterbar-item';
+import Notifications from 'notifications';
+import store from 'store';
+
+export default React.createClass( {
+	displayName: 'MasterbarItemNotifications',
+
+	propTypes: {
+		user: React.PropTypes.object,
+		isActive: React.PropTypes.bool,
+		className: React.PropTypes.string,
+		onClick: React.PropTypes.func,
+	},
+
+	getInitialState() {
+		let newNote = false;
+		let user;
+
+		if ( this.props.user ) {
```

Comment:
I thought it would be better with the idea of making the component more isolated/reusable, but it's correct that right now it's already checked in the parent. I'm neutral. Should I take it out?